company_default_partner_pricelist/migrations/13.0.1.0.1/post-migration.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2023 ForgeFlow, S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    company = env["res.company"]
```

Comment:
company_obj or ResCompany

Reply:
@hoangtrann These PRs are migrations that should not affect code refactoring. i think it is not necessary to make these changes as part of this PR